### ISSUE TITLE
v0.10.1: Missing argument for parameter idempotencyLevel in call... 

### ISSUE THREAD

**Issue by mycroftcanner** [connectrpc/connect-swift#224]
I get a bunch of these `Missing argument for parameter idempotencyLevel in call`  errors in my service.connect I generated with the latest buf. 

What do I have to do to use 0.10.1? Thanks

```
version: v1
plugins:
  - plugin: go
    out: gen
    opt: paths=source_relative
  - plugin: connect-go
    out: gen
    opt: paths=source_relative
  - plugin: buf.build/bufbuild/connect-swift
    opt: >
      GenerateAsyncMethods=true,
      GenerateCallbackMethods=true,
      Visibility=Public
    out: Sources
  - plugin: buf.build/apple/swift
    opt: Visibility=Public
    out: Sources%    
```

**Comment by rebello95** [comment]
You'll need to re-generate your sources using v0.10.1 of the plugin which was made available in https://github.com/bufbuild/plugins/pull/914.

`buf` should automatically use that if you pass `- plugin: bufbuild/connect-swift` in your `buf.gen.yaml`.

**Comment by zboralski** [comment]
@rebello95 I did and it is pointing to `- plugin: buf.build/bufbuild/connect-swift`. I tried `- plugin: bufbuild/connect-swift` as you said and it fails.

It doesn't seem to update to `v0.10.1` after generating new sources.

**Comment by mycroftcanner** [comment]
> You'll need to re-generate your sources using v0.10.1 of the plugin which was made available in [bufbuild/plugins#914](https://github.com/bufbuild/plugins/pull/914).
> 
> `buf` should automatically use that if you pass `- plugin: bufbuild/connect-swift` in your `buf.gen.yaml`.

Nope I regenerated the sources and still get the same error.

**Comment by rebello95** [comment]
Ah it looks like the plugin name changed in https://github.com/bufbuild/plugins/pull/914, and it's now `buf.build/connectrpc/swift`. Using `bufbuild/connect-swift` meant it was referencing an old version still, hence the lack of changes in your generated code.

If you use this, it should work (I tried it locally): `- plugin: buf.build/connectrpc/swift:v0.10.1`.

The docs are also updated/correct: https://connectrpc.com/docs/swift/getting-started/
